src/bindings/js/node/tests/unit/utils.js

```diff
@@ -2,6 +2,7 @@
 // Copyright (C) 2018-2024 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 
+const { addon: ov } = require('../..');
```

Comment:
Why?

Reply:
I thought it would be handy in comparison function but it hasn't been necessary so far. I'll delete it.